wire-schema/src/jvmMain/java/com/squareup/wire/schema/Extend.kt

```diff
@@ -37,9 +40,16 @@ class Extend private constructor(
     }
   }
 
-  fun validate(linker: Linker) {
+  fun validate(
+    linker: Linker,
+    syntax: Syntax?
+  ) {
     val linker = linker.withContext(this)
     linker.validateImport(location, type!!)
+
+    if (syntax === PROTO_3 && GOOGLE_PROTOBUF_OPTION_TYPES.none { it.toString() == name }) {
```

Comment:
I just thought `none {...}` is performant since we don't create a `ProtoType` instance every time we call `validate(...)`. With that said, I'll follow Kotlin-way :) > `ProtoType.get(name) !in GOOGLE_PROTOBUF_OPTION_TYPES`